.readthedocs.yml

```diff
@@ -20,4 +20,4 @@ formats: all
 python:
   version: 3.7
   install:
-    - requirements: requirements_dev.txt
+    - requirements: requirements.txt
```

Comment:
Once your PR is merged, please confirm that the docs build (or poke me to confirm this).